src/pipelines_repository/adf-build/target.py

```diff
@@ -115,4 +118,7 @@ def fetch_accounts_for_target(self):
         if (str(self.path)).startswith('/'):
             return self._target_is_ou_path()
 
+        if self.path is None:
```

Comment:
As per comment in stub_deployment_map - we don't have a j2 template to do cross region builds, nor has it been tested yet.